pkg/cosign/kubernetes/webhook/clusterimagepolicy/clusterimagepolicy_types.go

```diff
@@ -37,7 +50,7 @@ type Authority struct {
 	// +optional
 	Key *KeyRef `json:"key,omitempty"`
 	// +optional
-	Keyless *v1alpha1.KeylessRef `json:"keyless,omitempty"`
```

Comment:
suggestion: I am considering whether we should create again all the resources again. As a consequence, we should create the type for ImagePattern, Sources and Identities here to be consistent.

Reply:
@vaikas wdyt?

Reply:
Consistency is good indeed :) So, I'd be down for recreating them here, or always use one from the CRD.

Reply:
I kept the other ones as imports from the CRD because we arent currently manipulating it at the moment and it is a direct 1:1 mapping. 
We actually manipulate keyRefs so that is why Authorities and key[less]Refs were re-implemented as required.

When the other properties diverge, I feel that updating it then makes more sense to keep potential updates in one place as Ville mentions until that diverging.